rsmanage/rsmanage.py

```diff
@@ -393,9 +393,15 @@ def build(config, clone, ptx, gen, manifest, course):
                 )
                 exit(1)
 
-    # proj_dir = os.path.basename(repo).replace(".git", "")
+    # if course name is different from book name, clone to the right folder
+    if not os.path.isdir(course):
```

Comment:
This is a problem... The course name has already been checked to be sure it is in the database.  So you can't just change it now to match what was on the github url.

It would be better to change line 389 to have the clone command always clone the url into the name of the course.

Reply:
Actually, come to think of it, wouldn't this make the course name flag obsolete (as it just copies to the folder anyways)?

Reply:
No, because we need to verify that the course is in the database.

Reply:
Alright, I've changed it to clone into the course name directory, however, this causes this error:
`Error: Example and thinkcspy do not match.  Your course name needs to match the project_name in pavement.py
Build Failed, see cli.log for details`

Do I need to change more than just the clone command? I'm starting to wonder if I was just cloning wrong originally.

Reply:
I think the first method was where I was going with this. Is pavement.py edited anywhere to begin with or do I need to write new code for that? I've written some code for cloning to the correct directory.